lib/cc/analyzer/filesystem.rb

```diff
@@ -47,6 +37,12 @@ def files_matching(globs)
 
       private
 
+      def file_paths
+        @file_paths ||= Dir.chdir(@root) do
+          `find . -type f`.strip.split("\n")
```

Comment:
@pbrisbin `find` is not intended to be a speed improvement, it's an improvement because `Dir.glob` blocks signal handling, and so you can't stop it with CTRL+C.

I will use `-print0`